app/Mage.php

```diff
@@ -444,7 +444,7 @@ public static function getStoreConfigFlag($path, $store = null)
     /**
      * Get base URL path by type
      *
-     * @param string $type
+     * @param Mage_Core_Model_Store::URL_TYPE_* $type
```

Comment:
I have not seen this before, not sure if that's a valid doc block. Alternative:
```suggestion
     * @param string $type One of Mage_Core_Model_Store::URL_TYPE_* constants
```

Reply:
It seems that it is supported https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants

Reply:
Thanks, learn something new.